src/Symfony/Component/Validator/Constraints/Negative.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+class Negative extends LessThan
+{
+    public $message = 'This value should be negative.';
+
+    public function __construct($options = null)
+    {
+        if (isset($options['propertyPath'])) {
```

Comment:
i think we need to account for is_array as well :)

https://3v4l.org/U91Ee

Reply:
`s/isset/array_key_exists/`?

Reply:
oh yes, thanks for the hint :-)